src/components/symbols/OSymbol.tsx

```diff
@@ -23,6 +23,12 @@ export class OSymbol extends React.Component<OSymbolProps, OSymbolState> {
         super( props );
     }
 
+    shouldComponentUpdate(nextProps: OSymbolProps) {
+        const differentSize = this.props.bigSymbol !== nextProps.bigSymbol;
+        const differentStyle = this.props.style !== nextProps.style;
```

Comment:
Why aren't you checking `shouldAnimate`?

Reply:
I didn't check that because it doesn't change.